### PR TITLE
[DX-673]linking between Product Stack and Developer Support Sections For Previous Bundled Release Notes

### PR DESCRIPTION
**Pull request by dcs3spp**
[DX-673](https://tyktech.atlassian.net/browse/DX-673) relates to [DX-615](https://tyktech.atlassian.net/browse/DX-615)

Create linking pages between Product Stack and Developer Support For Previous Bundled Release Notes

[GW older release notes preview](https://deploy-preview-3227--tyk-docs.netlify.app/docs/nightly/product-stack/tyk-gateway/release-notes/overview)

[Dashboard older release notes preview](https://deploy-preview-3227--tyk-docs.netlify.app/docs/nightly/product-stack/tyk-dashboard/release-notes/overview)

[Developer Support Section preview](https://deploy-preview-3227--tyk-docs.netlify.app/docs/nightly/release-notes)

[DX-673]: https://tyktech.atlassian.net/browse/DX-673?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[DX-615]: https://tyktech.atlassian.net/browse/DX-615?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/overview.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+title: Tyk Gateway Release Notes
+description: "Explains where to access older supported bundled release notes for Tyk Dashboard and Tyk Gateway"
+tags: ["Release notes"]
+---
+Older release notes of Tyk Gateway were bundled together with Tyk Dashboard. The release notes for these older supported releases are available in the [Developer Support]({{< ref "/developer-support/tyk-release-summary/overview" >}}) section of the website.
```
Comment (letzya) [inline tyk-docs/content/product-stack/tyk-gateway/release-notes/overview.md:6]:
```suggestion
Prior to the 5.2 release, Tyk Gateway release notes were combined with those of Tyk Dashboard. You can find these combined release notes in the [Developer Support]({{< ref "/developer-support/tyk-release-summary/overview" >}}) section of the website.

For Tyk Gateway release notes v5.2 and beyond, please check this directory.
```

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/overview.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+title: Tyk Dashboard Release Notes
+description: "Explains where to access older supported bundled release notes for Tyk Dashboard and Tyk Gateway"
+tags: ["Release notes"]
+---
+Older release notes of Tyk Dashboard were bundled together with Tyk Gateway. The release notes for these older supported releases are available in the [Developer Support]({{< ref "/developer-support/tyk-release-summary/overview" >}}) section of the website.
```
Comment (letzya) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/overview.md:6]:
```suggestion
```suggestion
Prior to the 5.2 release, Tyk Dashboard release notes were combined with those of Tyk Gateway. You can find these combined release notes in the [Developer Support]({{< ref "/developer-support/tyk-release-summary/overview" >}}) section of the website.

For Tyk Dashboard release notes v5.2 and beyond, please check this directory.
```

File: tyk-docs/content/developer-support/tyk-release-summary/overview.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+title: Release Notes
```
Comment (letzya) [inline tyk-docs/content/developer-support/tyk-release-summary/overview.md:2]: @dcs3spp please check this page https://deploy-preview-3227--tyk-docs.netlify.app/docs/nightly/release-notes/  - do you think we need both?

Comment (dcs3spp) [reply tyk-docs/content/developer-support/tyk-release-summary/overview.md:2]: @letzya Would it be ok to replace the content of https://deploy-preview-3227--tyk-docs.netlify.app/docs/nightly/release-notes/ with the new content from overview.md in this PR

Comment (dcs3spp) [reply tyk-docs/content/developer-support/tyk-release-summary/overview.md:2]: @letzya I can move the new content of the overview file into [release notes file](https://deploy-preview-3227--tyk-docs.netlify.app/docs/nightly/release-notes/)

Comment (letzya) [reply tyk-docs/content/developer-support/tyk-release-summary/overview.md:2]: what do you think is best? which to move into which?

Comment (dcs3spp) [reply tyk-docs/content/developer-support/tyk-release-summary/overview.md:2]: @letzya I have pushed the changes up into the PR. I have removed the overview file and moved it's content into the release notes file.

Comment (dcs3spp) [reply tyk-docs/content/developer-support/tyk-release-summary/overview.md:2]: @letzya  I have made the changes to the PR. The release notes page in developer support is now replaced with an overview page in the new content folder for _/developer-support_ section. An alias has been added to the overview page.

**Comment by dcs3spp** [comment]
@yaara This one requires reviewing again relating to release notes

**Review by letzya — APPROVED** [review]

**Comment by letzya** [comment]
Love this @dcs3spp !

**Pull request merged**